source/scripting_v3/GTA/Blip.cs

```diff
@@ -111,6 +289,106 @@ public float Scale
 			set => Function.Call(Hash.SET_BLIP_SCALE, Handle, value);
 		}
 
+		/// <summary>
+		/// Gets or sets the x-axis scale of this <see cref="Blip"/> on the map.
+		/// </summary>
+		public float ScaleX
+		{
+			get
+			{
+				var address = MemoryAddress;
+				if (address == IntPtr.Zero)
+				{
+					return 0;
+				}
+
+				return SHVDN.NativeMemory.ReadFloat(address + 0x50);
+			}
+			set
+			{
+				var address = MemoryAddress;
+				if (address == IntPtr.Zero)
+				{
+					return;
+				}
+
+				SHVDN.NativeMemory.WriteFloat(address + 0x50, value);
+			}
+		}
+
+		/// <summary>
+		/// Gets or sets the y-axis scale of this <see cref="Blip"/> on the map.
+		/// </summary>
+		public float ScaleY
+		{
+			get
+			{
+				var address = MemoryAddress;
+				if (address == IntPtr.Zero)
+				{
+					return 0;
+				}
+
+				return SHVDN.NativeMemory.ReadFloat(address + 0x54);
+			}
+			set
+			{
+				var address = MemoryAddress;
+				if (address == IntPtr.Zero)
+				{
+					return;
+				}
+
+				SHVDN.NativeMemory.WriteFloat(address + 0x54, value);
+			}
+		}
+
+		/// <summary>
+		/// Gets or sets the interval in ms between each blip flashing.
+		/// The value is up to 65535.
+		/// </summary>
+		public int FlashInterval
+		{
+			get
+			{
+				var address = MemoryAddress;
+				if (address == IntPtr.Zero)
+				{
+					return 0;
+				}
+
+				return SHVDN.NativeMemory.ReadInt16(address + 0x44);
+			}
+			set => Function.Call(Hash.SET_BLIP_FLASH_INTERVAL, Handle, value);
+		}
+
+		/// <summary>
+		/// Gets or sets the interval in ms between each blip flashing.
+		/// The value is up to 65534.
+		/// </summary>
+		/// <remarks>returns <c>-1</c> if the internal value of this property value is set to <c>65535</c>, which indicates that the flashing timer is not set.</remarks>
+		public int FlashTimer
```

Comment:
Do you know what the difference between `FlashInterval` and `FlashTimer` is? It's not really clear to me.

Reply:
`FlashTimer` determines how long blips flash before they stops flashing. And the low `FlashInterval` value is, the shorter time blips flash once. Any better name suggestions?

Reply:
Maybe `FlashDuration`?

Reply:
I think `Blip.FlashTimeLeft` is also more clear than `Blip.FlashTimer`. We've already had `Vehicle.AlarmTimeLeft`.

Reply:
Sounds good!